unstable.txt

```diff
@@ -1,8 +1,7 @@
-ddl TestTestSerialStatSuite
-ddl testSerialDBSuite.TestModifyColumnTypeWhenInterception
-ddl TestStateChange
-ddl testDBSuite1.TestAddIndexWithSplitTable
+ddl TestModifyColumn
 util/cpuprofile TestGetCPUProfile
 executor TestDefaultValForAnalyze
-executor TestPBMemoryLeak
-executor TestDirectReadingWithAgg
+executor TestInvalidReadCacheTable
+executor TestInvalidReadTemporaryTable
```

Comment:
I have a PR for this one https://github.com/pingcap/tidb/pull/32959
Let's see does it work.